apps/src/ailab/Ailab.js

```diff
@@ -31,8 +32,11 @@ function getInstructionsDefaults() {
       'Choose one or more columns as inputs to help make the prediction.',
     selectTrainer: 'Set up the training.',
     trainModel: 'Your model is being trained.',
+    modelTraining:
+      'Your model is undergoing training.  Notice how we do one row at a time!',
     results: 'Review the results.',
     saveModel: 'Save the trained model for use in App Lab.',
+    trainingSettings: 'Adjust column types here if you want.',
```

Comment:
Can we track or add a comment about removing this line when main-next is merged since the training settings screen no longer exists? (https://github.com/code-dot-org/ml-playground/pull/178)

Reply:
Good point.  Given that we're about to reach that point, I just removed the unnecessary strings.